lib/stack_master/sparkle_formation/user_data_file.rb

```diff
@@ -0,0 +1,32 @@
+require 'sparkle_formation'
+require 'erubis'
+
+class SparkleFormation
+  module SparkleAttribute
+    module Aws
+      class SfEruby < Erubis::Eruby
+        include Erubis::ArrayEnhancer
+
+        def add_expr(src, code, indicator)
+          case indicator
+            when '='
+              src << " #{@bufvar} << (" << code << ');'
```

Comment:
That breaks the test for custom vars, instead of returning the variables value it returns true.

```
       -"Fn::Base64" => {"Fn::Join"=>["", ["#!/bin/bash\n", "test_var", "\n", "yes", "\n"]]},
       +"Fn::Base64" => {"Fn::Join"=>["", ["#!/bin/bash\n", "test_var", "\n", true, "\n"]]},
```